src/keystone/keystone_shim.cc

```diff
@@ -0,0 +1,346 @@
+#include "keystone_api.h"
+#include "certifier_framework.h"
+#include "certifier_utilities.h"
+#include <string.h>
+
+using std::string;
+using namespace certifier::utilities;
+
+
+//  Copyright (c) 2021-22, VMware Inc, and the Regents of the University of California.
```

Comment:
This shim file may go away eventually ... but, meanwhile, for consistency, can you move this to the top of the file?